frontend/src/e2e-test/specs/5_employee/child-documents.spec.ts

```diff
@@ -89,36 +90,110 @@ describe('Employee - Child documents', () => {
     // End of admin creates a template
 
     // Unit supervisor creates a child document
-
     page = await Page.open({ mockedTime: now.toSystemTzDate() })
     await employeeLogin(page, unitSupervisor)
     await page.goto(
       `${config.employeeUrl}/child-information/${childFixture.id}`
     )
-    const childInformationPage = new ChildInformationPage(page)
+    let childInformationPage = new ChildInformationPage(page)
     let childDocumentsSection = await childInformationPage.openCollapsible(
       'childDocuments'
     )
     await childDocumentsSection.createDocumentButton.click()
+    await childDocumentsSection.createModalTemplateSelect.assertTextEquals(
+      'HOJKS 2022-2023'
+    )
     await childDocumentsSection.modalOk.click()
 
-    const childDocument = new ChildDocumentPage(page)
+    // Fill an answer and return
+    let childDocument = new ChildDocumentPage(page)
+    await childDocument.status.assertTextEquals('Luonnos')
     const answer = 'Jonkin sortin vastaus'
     let question = childDocument.getTextQuestion(sectionName, questionName)
     await question.fill(answer)
     await childDocument.savingIndicator.waitUntilHidden()
     await childDocument.previewButton.click()
     await childDocument.returnButton.click()
 
+    // Assert status draft and unpublished, open the document again
     childDocumentsSection = await childInformationPage.openCollapsible(
       'childDocuments'
     )
     await waitUntilEqual(childDocumentsSection.childDocumentsCount, 1)
-    const row = childDocumentsSection.childDocuments(0)
+    let row = childDocumentsSection.childDocuments(0)
     await row.status.assertTextEquals('Luonnos')
+    await row.published.assertTextEquals('-')
     await row.openLink.click()
+    const documentUrl = page.url
 
+    // Assert answer was saved
     question = childDocument.getTextQuestion(sectionName, questionName)
     await question.assertValueEquals(answer)
+
+    // Publish without changing status
+    await childDocument.previewButton.click()
+    await childDocument.publish()
+
+    // Assert status and publish time
+    await childDocument.returnButton.click()
+    childDocumentsSection = await childInformationPage.openCollapsible(
+      'childDocuments'
+    )
+    await waitUntilEqual(childDocumentsSection.childDocumentsCount, 1)
+    row = childDocumentsSection.childDocuments(0)
```

Comment:
Not relevant for this PR, but I think having  a `childDocumentsSection.childDocumentRows` locator would be a better abstraction. You could then just have:

```
await childDocumentsSection.childDocumentRows.assertCount(1)
row = childDocumentsSection.childDocumentRows.nth(0)
```